website/sidebars.js

```diff
@@ -4,9 +4,188 @@
 const sidebars = {
   // Generate sidebar from the docs folder (or versioned_docs/<version>)
   mySidebar: [
+    'intro',
     {
-      type: 'autogenerated',
-      dirName: '.',
```

Comment:
Does this mean that we would need to manually add entries to the navigation bar? I'd prefer not :/